docs/content/Link.mdx

```diff
@@ -10,10 +10,6 @@ import data from '../../src/Link/Link.docs.json'
 
 The Link component styles anchor tags with default hyperlink color cues and hover text decoration. `Link` is used for destinations, or moving from one page to another.
 
-In special cases where you'd like a `<button>` styled like a `Link`, use `<Link as='button'>`. Make sure to provide a click handler with `onClick`.
-
-**Important:** When using the `as` prop, be sure to always render an accessible element type, like `a`, `button`, `input`, or `summary`.
-
```

Comment:
With this change, could we update the [props table](https://primer-208e588ead-13348165.drafts.github.io/Link#link) in the docs and remove the `as` prop entirely? I believe this is in `Link.docs.json`. We might also need to do the same with `LinkItem` since we're removing usage of `as` there too.

Reply:
Good call.